packages/ui/src/Integration/IntegrationDetailTab.tsx

```diff
@@ -0,0 +1,71 @@
+import { Button, Grid, Icon, ListView, ListViewItem } from 'patternfly-react';
+import * as React from 'react';
+import { IntegrationStepsHorizontalView } from './IntegrationStepsHorizontalView';
+
+export interface IIntegrationDetailTabProps {
+  description?: string;
+  integrationIsDraft: boolean;
+  i18nTextBtnEdit?: string;
+  i18nTextBtnPublish?: string;
+  i18nTextDraft?: string;
+  i18nTextHistory?: string;
+  steps?: any;
+}
+export class IntegrationDetailTab extends React.PureComponent<
+  IIntegrationDetailTabProps
+> {
+  public render() {
+    return (
+      <>
+        <div>
+          <IntegrationStepsHorizontalView steps={this.props.steps} />
+          <div>
+            {this.props.description ? (
+              <p>
+                {this.props.description}&nbsp;
+                <Icon name={'pencil'} />
+              </p>
+            ) : null}
+          </div>
+        </div>
+        <div>
```

Comment:
This is essentially where the `<IntegrationDetailHistoryListView />` should be. Right now it's just markup. I can change it, but then would need to pass an `integration` property here, unless you know of a better way?